test/syntax/idl/extended-attributes.webidl

```diff
@@ -7,7 +7,7 @@ interface ServiceWorkerGlobalScope : WorkerGlobalScope {
 
 // Conformance with ExtendedAttributeList grammar in http://www.w3.org/TR/WebIDL/#idl-extended-attributes
 // Section 3.11
-[IntAttr=0, FloatAttr=3.14, StringAttr="abc", IdentifierAttr=_null, IdentifiersAttr=(_null, _const)]
+[IntAttr=0, FloatAttr=3.14, StringAttr="abc", IdentifierAttr=_null, IdentifiersAttr=(_null, _const), FloatList=(3.14)]
```

Comment:
What would happen if we have a mixed attribute list? Like `[List=(8, 8.5)]`? I’d assume this should get parsed as a decimal-list but just wanted to contribute the test case.

Reply:
Do we have a use case for mixed lists? The current implementation prevents mixed list as no such use case has been observed. For your specific example, I would suggest `[List=(8.0, 8.5)]`.

Sorry to respond too late 😥

Reply:
Your suggestion sounds fine – we'll report back if we happen to need mixed lists :wink:. Thanks for the clarification!